moma_mission/config/state_machine/piloting_sim.yaml

```diff
@@ -0,0 +1,32 @@
+OPEN_GRIPPER:
+  default_outcome: None
+  gripper_action_name: /control/franka_gripper/gripper_action
+  position: 0.03
+  max_effort: 20
+  tolerance: 0.001
+  timeout: 5.0
+
+CLOSE_GRIPPER:
+  default_outcome: None
+  gripper_action_name: /control/franka_gripper/gripper_action
+  position: 0.0
+  max_effort: 20
+  tolerance: 0.03
+  timeout: 5.0
+
+APPROACH_VALVE:
+  default_outcome: None
+  manager_namespace: control
+  startlist: ['mpc_controller', 'path_passthrough_controller']
+  stoplist: ['effort_joint_trajectory_controller']
+
+  turning_angle_deg: 0.0
+
+MANIPULATE_VALVE:
+  default_outcome: None
+  manager_namespace: control
+  startlist: ['mpc_controller', 'path_passthrough_controller']
+  stoplist: ['effort_joint_trajectory_controller']
+
+  turning_angle_deg: 120.0
+  turning_speed_deg: 20.0
```

Comment:
Do we need a separate sim state machine? I would try to enforce as much as possible interoperability. I know this is some effort and also that a separate sim pipeline helps to get things done faster but with a single state machine file we are forced to reproduce as closely as possible the real experiments. 

I am not sure why we have this here but I assume is for testing the valve grasping. This is indeed a good featue to have on the real robot as well whenever we want to test manipulation given a a priori known pose of the valve (from previous experiment for example)